# ValueError was raised for math.log(NAN)

import math
print(math.log(float('nan')))
